src/test/java/seedu/address/model/borrower/BorrowerIdGeneratorTest.java

```diff
@@ -19,6 +21,9 @@ void generateBorrowerId_autoGenerateSuccess() {
         assertEquals(BorrowerIdGenerator.generateBorrowerId(), new BorrowerId("K0002"));
         assertEquals(BorrowerIdGenerator.generateBorrowerId(), new BorrowerId("K0003"));
         assertEquals(BorrowerIdGenerator.generateBorrowerId(), new BorrowerId("K0004"));
+        borrowers.addBorrower(new BorrowerBuilder(ALICE).withBorrowerId("K0005").build());
```

Comment:
We should give `ALICE` and `BOB` non-consecutive serial numbers (Like `K0100` and `K0006`) to test that the generator will generate the next valid one correctly (which should be `K0005` followed by `K0007`)

Reply:
Good suggestion!